docs/objects.md

```diff
@@ -328,6 +328,195 @@ The formula to derive the pool's PKIprofile is as follows:
 ```
 pkiprofilenam = poolName + "-pkiprofile"
 ```
+### Markers for Avi Objects
+
+With Avi controller version 20.1.5, Avi objects can be labelled with markers. Each marker is pair of `key` : `<value1>`, `<value2>`. Each object can have multiple markers. Markers can be used to group objects at Avi controller side and provide permission to users to access these objects. Avi controller support markers to all license types from 20.1.6.
```

Comment:
Each marker is a

Reply:
Markers are used to group/filter objects on the Avi Controller and can be utilised to provide user access permissions using the GRBAC feature on the Avi Controller.

Avi controller supports markers in all controller license types from 20.1.6.